x-pack/plugins/apm/server/tutorial/instructions/apm_agent_instructions.ts

```diff
@@ -457,8 +457,8 @@ export const createDotNetAgentInstructions = (
       Core download the [Elastic.Apm.NetCoreAll]({netCoreAllApmPackageLink}) package. This package will automatically add every \
       agent component to your application. \n\n In case you would like to minimize the dependencies, you can use the \
       [Elastic.Apm.AspNetCore]({aspNetCorePackageLink}) package for just \
-      ASP.NET Core monitoring or the [Elastic.Apm.EfCore]({efCorePackageLink}) package for just Entity Framework Core monitoring. \n\n \
-      In case you only want to use the public Agent API for manual instrumentation use the [Elastic.Apm]({elasticApmPackageLink}) package.',
+      ASP.NET Core monitoring or the [Elastic.Apm.EfCore]({efCorePackageLink}) package for just Entity Framework Core monitoring. \n\n In \
+      case you only want to use the public Agent API for manual instrumentation use the [Elastic.Apm]({elasticApmPackageLink}) package.',
```

Comment:
This break at new line was confusing the markdown parse of `EuiMarkdownFormat`, which i replaced with `kibana_react`'s `Markdown` component since it supports `target="_blank"` in links.